nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -110,11 +152,32 @@ def _iter_signals_from_spheres(seeds, niimg, radius, allow_overlap,
                                         allow_overlap,
                                         mask_img=mask_img)
     for i, row in enumerate(A.rows):
-        if len(row) == 0:
-            raise ValueError('Sphere around seed #%i is empty' % i)
         yield X[:, row]
 
 
+def _iter_regions_from_spheres(seeds, radius, allow_overlap, mask_img):
+    """Utility function to iterate over spheres. Used in NiftiSpheresMasker()
+    .inverse_transform()
+
+    Parameters
+    ----------
+    seeds: List of triplets of coordinates in native space
+        Seed definitions. List of coordinates of the seeds in the same space
+        as target_affine (either of mask or if mask is None MNI space).
+    radius: float
+        Indicates, in millimeters, the radius for the sphere around the seed.
+    allow_overlap: boolean
+        If False, an error is raised if the maps overlaps (ie at least two
+        maps have a non-zero value for the same voxel).
+    mask_img: Niimg-like object,
+        Mask as a reference space to project the spheres back into brain space.
+    """
+    _, adjacency = _apply_mask_and_get_affinity(seeds, None, radius,
+                                                allow_overlap,
+                                                mask_img=mask_img)
+    yield from adjacency.rows
```

Comment:
Where is this function used ?

Reply:
it isn't :) forgot to remove it after some refactoring